x-pack/plugins/osquery/public/live_queries/form/packs_combobox_field.tsx

```diff
@@ -146,15 +127,15 @@ export const PacksComboBoxField = ({ field, euiFieldProps = {}, idAria, ...rest
         })}
         selectedOptions={selectedOptions}
         onChange={handlePackChange}
-        onSearchChange={onSearchComboChange}
+        // onSearchChange={onSearchComboChange} // TODO DO WE STILL NEED THIS?
```

Comment:
I think we should allow filtering results, what do you think?

Reply:
filtering works fine. this one just used to clear errors on the field If I am not wrong... 
I believe we do not need it any more